tools/releases/docker.sh

```diff
@@ -15,7 +15,13 @@ function build() {
   for component in ${KUMA_COMPONENTS}; do
     for arch in ${BUILD_ARCH}; do
       msg "Building $component..."
-      docker build --build-arg ARCH="${arch}" --build-arg ENVOY_VERSION="${ENVOY_VERSION}" -t "${KUMA_DOCKER_REPO_ORG}/${component}:${KUMA_VERSION}-${arch}" \
+      base_image_arch=$arch
+      # ARM base images have different tags
+      if [ "$component" != "kuma-dp" ] && [ "$arch" == "arm64" ]; then
+        base_image_arch="arm64v8"
+      fi
+      echo $base_image_arch
+      docker build --build-arg ARCH="$arch" --build-arg BASE_IMAGE_ARCH="$base_image_arch" --build-arg ENVOY_VERSION="${ENVOY_VERSION}" -t "${KUMA_DOCKER_REPO_ORG}/${component}:${KUMA_VERSION}-${arch}" \
```

Comment:
We have tags per arch? I thought this wasn't required but I could be very wrong :)

Reply:
We need to publish first images per arch and later step `manifest` aggregates them into one tag.